lib/server.go

```diff
@@ -1859,6 +2006,17 @@ func (srv *Server) _handleBlock(pp *Peer, blk *MsgDeSoBlock) {
 	glog.Infof(CLog(Cyan, fmt.Sprintf("Server._handleBlock: Received block ( %v / %v ) from Peer %v",
 		blk.Header.Height, srv.blockchain.headerTip().Height, pp)))
 
+	// If the node is running a Fast-HotStuff validator and the consensus is running,
+	// in the steady-state, then we handle the block according to the consensus rules.
+	if srv.fastHotStuffConsensus != nil && srv.fastHotStuffConsensus.IsRunning() {
+		srv.fastHotStuffConsensus.HandleBlock(pp, blk)
```

Comment:
I notice you're ignoring the error here. What it looks like is happening right now is the following:
0. When we boot up the node before the PoS cut-over height, we will set its PoS params, which will cause us to enter this loop (note that IsRunning() doesn't have a block height check).
1. Before the PoS cut-over height, we are calling fasthotStuffConsensus.HandleBlock() -> processBlockPoS -> tryConnectBlockAsNewTip (or whatever it's called), which is erroring because we're not at the cut-over height yet.
2. The error bubbles all the way up to here, where it's ignored.
3. Eventually, we pass the cut-over height and this stops erroring.

Instead, I recommend the following:
1. Gate this on the cut-over height so we don't enter it until we're supposed to.
2. Catch the error here, since it probably means something is messed up.

Reply:
Sounds good. I'll catch and log the error here

Reply:
Resolved in PR https://github.com/deso-protocol/backend/pull/538